ajax/libs/urlive/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "urlive",
+  "filename": "jquery.urlive.min.js",
+  "version": "1.1.1",
+  "description": "jQuery URLive lets you easily create a live preview of a URL.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/markserbol/urlive"
+  },
+  "keywords": [
+    "url",
+    "preview",
+    "jquery",
+    "facebook",
+    "snippet"
+  ],
+  "author": "Mark Serbol <markserbol@outlook.com>",
+  "license": "MIT"
+}
```

Comment:
Let's add two filed as the followings:
```js
  "homepage": "https://markserbol.com/urlive/",
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/markserbol/urlive.git",
    "fileMap": [
      {
        "basePath": "",
        "files": [
          "jquery.urlive*.+(js|css|map)"
        ]
      }
    ]
  }
```

1. It will useful to people to add `homepage` and let people know more information about this library.
2. Adding auto-update config is better when new version release can let CDNJS host newest version od this library.